notebooks/dataframes/struct_and_array_dtypes.ipynb

```diff
@@ -0,0 +1,658 @@
+{
```

Comment:
<p>Nit: "Let's"</p><p> </p><p>Suggestion:</p><p> </p><p>Create a DataFrame with an <code>Address</code> struct column by using dictionaries for the data and setting the dtype to <code>struct&lt;City: string, State: string&gt;[pyarrow]</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/883/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Thanks for suggestions!</p><p> </p><p> </p><p> </p><p> </p>